chalice/package.py

```diff
@@ -798,17 +803,11 @@ def _generate_restapi(self, resource, template):
                         'minimum_compression_size'] = int(
                             resource.minimum_compression)
 
-        template['resource'].setdefault('aws_api_gateway_stage', {})[
-            resource.resource_name] = {
-                'rest_api_id': '${aws_api_gateway_rest_api.%s.id}' % (
-                    resource.resource_name),
-                'stage_name': resource.api_gateway_stage,
-                'deployment_id': '${aws_api_gateway_deployment.%s.id}' % (
-                    resource.resource_name)
-        }
-
         template['resource'].setdefault('aws_api_gateway_deployment', {})[
             resource.resource_name] = {
+                'stage_name': resource.api_gateway_stage,
+                'stage_description': (
```

Comment:
Actually it might be worth to put a comment on why we are adding the md5 to the stage description.

Reply:
done.